src/project-3.js

```diff
@@ -1,83 +1,109 @@
 // Do not change any of the function names
 
 const makeCat = (name, age) => {
+  const newObj = { name, age, meow() { return 'Meow!'; } };
```

Comment:
You can also do: 
```
{
  meow: () => 'Meow!'
}
```
The difference between the two approaches is that the method shorthand you used binds `this` and the arrow function method doesn't